src/Radarr.Api.V3/CustomFormats/FormatTagValidator.cs

```diff
@@ -24,7 +24,7 @@ protected override bool IsValid(PropertyValidatorContext context)
 
             var invalidTags = tags.Where(t => !FormatTag.QualityTagRegex.IsMatch(t));
 
-            if (invalidTags.Count() == 0)
+            if (invalidTags.Any())
```

Comment:
This should be `!invalidTags.Any()`. The format is valid if there are no `invalidTags`. Currently one cannot add new tags as the validator always fails.